### ISSUE TITLE
Android Studio indexing all the time

### ISSUE THREAD

**Issue by DrivEddy** [flutter/flutter#15973]
Sorry if that is not the right place to post this issue! 

## Steps to Reproduce

After updating to the latest Android Studio and the latest Flutter plugin, the IDE start indexing every second for about half a second. Autocompletions and similar things stopping working for this moment. 

Flutter Plugin version 22.2.2

I tried to `File -> Invalidate Cache and Restart` but it has no effect 

## Flutter Doctor

```
[✓] Flutter (Channel master, v0.2.5-pre.44, on Mac OS X 10.13.3 17D102, locale en-DE)
    • Flutter version 0.2.5-pre.44 at /Users/eikebartels/Applications/flutter
    • Framework revision 067be92ae8 (8 hours ago), 2018-03-26 22:16:48 -0700
    • Engine revision dd6f46c485
    • Dart version 2.0.0-dev.41.0.flutter-2f68e82526

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/eikebartels/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.
```

**Comment by kagonzalez84** [comment]
Same happen to me

**Comment by zoechi** [comment]
I think this is a dup of flutter-intellij#1735

**Comment by DrivEddy** [comment]
https://github.com/flutter/flutter-intellij/issues/1735

**Comment by DrivEddy** [comment]
Seems to be solved in `23.1`
